lib/proforma/xml_generator.rb

```diff
@@ -0,0 +1,142 @@
+require 'nokogiri'
```

Comment:
this serves as the "exporter" ?

Reply:
Yes, nokogiri is a gem to generate and read xml data.